src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/FootnoteRepository.cs

```diff
@@ -182,7 +187,23 @@ private async Task DeleteFootnote(Guid releaseId, Footnote footnote, bool canRem
                 if (canRemoveFootnote)
                 {
                     var footnoteToRemove = await _context.Footnote.SingleAsync(f => f.Id == footnote.Id);
+                    var footnotesToReorder = await _context.ReleaseFootnote
+                        .Include(rf => rf.Footnote)
+                        .Where(rf => rf.ReleaseId == releaseId &&
+                                     rf.Footnote.Order > footnoteToRemove.Order)
+                        .Select(rf => rf.Footnote)
+                        .OrderBy(f => f.Order)
+                        .ToListAsync();
+
                     _context.Footnote.Remove(footnoteToRemove);
+                    _context.Footnote.UpdateRange(footnotesToReorder);
+                    footnotesToReorder.ForEach(footnoteToReorder =>
+                    {
+                        if (footnoteToReorder.Order > 0)
```

Comment:
Nice one - I was looking specifically to see if you were preventing existing ones from going negative if deleting footnotes when a bunch of them have the default order of zero, so nicely considered :)